cgi/generate_sample_import_file.pl

```diff
@@ -59,13 +59,21 @@
 	mandatory => $workbook->add_format(border => 1, bold => 1, bg_color => '#aaffcc'),
 	recommended => $workbook->add_format(border => 1, bold => 1, bg_color => '#ccffdd'),
 	optional => $workbook->add_format(border => 1, bold => 1, bg_color => '#eeffee'),
+	description => $workbook->add_format(italic => 1, text_wrap => 1, align => 'left', align => 'vcenter'),
 );
 
 # Re-use the structure used to output select2 options in import_file_select_format.pl
 my $select2_options_ref = generate_import_export_columns_groups_for_select2([$lc]);
 
 my $headers_row = 0;
-my $col = 0;
+my $description_row = 1;
+my $col = 1;
+
+my $field_id;
+my $comment;
+
+$worksheet->set_column('A:A', 30);
+$worksheet->write( $description_row, 0, "Description", $formats{'description'});
```

Comment:
Could you make "Description" a translatable string?

To do that, you need to put lang("description") instead, and then add a stringid "description" in po/common.po and po/en.po (and it would be good to add it in another language so that you can test that you get a different string if you load for instance the site in Spanish: http://es.openfoodfacts.localhost/cgi/generate_sample_import_file.pl

Once you have added / changed a string in po/common.pot / en.po / es.po , you need to run:
make build_lang
make restart

and then you can use lang("description") in the Perl file instead of "Description".

Reply:
Hi @stephanegigandet, 

Yes, I can. But I don't understand if I should add it as stringid or as msid, like in the po/common.po file?

<img width="481" alt="Screenshot 2024-03-15 at 15 10 12" src="https://github.com/openfoodfacts/openfoodfacts-server/assets/24766978/1face8ae-bc93-431e-a073-fda8897b9a7b">

Reply:
Followed the instructions and turned "Description" into a translatable string

Reply:
@anaritadauane Great :)

Can you also add this to en.po so that we have an English translation (English is the default):


```
msgctxt "description"
msgid "Description"
msgstr "Description"
```

Reply:
Done